openblas-build/src/build.rs

```diff
@@ -116,6 +117,99 @@ pub enum Target {
     Z14,
 }
 
+impl FromStr for Target {
```

Comment:
I would probably generate both the enum and this impl via a macro to avoid repetition. But if cumbersome, let’s keep it as is.